reVX/least_cost_xmission/layers/layer_creator.py

```diff
@@ -130,7 +168,7 @@ def _process_raster_layer(self, data: npt.NDArray,  # type: ignore[return]
 
             return processed
 
-        # No range, has to be map. Assign cells values based on map.
+        # No bins specified, has to be map. Assign cells values based on map.
```

Comment:
Is it possible to end up here by user error when map is not intended?

Reply:
Yea good question. Let's see...

`forced_inclusion is not None` is handled outside of this function, as is `rasterize is not None`.

`global_value is not None` returns within this function above, and so does `bins is not None` does the same.

I think the only possible issue here may be if none of the required config keys are set, but the CLI checks for this. So as long as you are always using the CLI, this should be perfectly ok. 

Anything I miss?